worker/manager/src/mirrors_qa_manager/worker.py

```diff
@@ -0,0 +1,309 @@
+# pyright: strict, reportMissingTypeStubs=false, reportUnknownMemberType=false, reportOptionalSubscript=false, reportUnknownVariableType=false, reportUnknownArgumentType=false
```

Comment:
Wouldn't having it in the `pyproject` cause it to skip checking all the code? I think why the errors throw up is because `docker` lacks a stub file. And having those turned off only for code that uses docker would be better than for the whole project.